cpp/ps0000/problem 0.cpp

```diff
@@ -1,14 +1,15 @@
-#include <iostream>
+#include <iostream> // 2/2
 // first mistake : @ ,, # not #
 // second mistake : "iostream.h" ,, should be iostream without .h and ""
 using namespace std;
 //forgot this line
-int main()
-// he write void and we don't need it ,, Main instead main
+int main() // 1/1
+// Main instead main
 {
-  cout << "Hello World \n " ;
+  cout << "Hello, World\n"; // 3/4
 // first mistake >> ,, << not >>
 // second mistake /n ,, \n not /n

```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [x] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [ ] output is as required `Hello, World!` in a single line